.github/dependabot.yml

```
@@ -3,4 +3,14 @@
-      interval: "weekly"
+      interval: "monthly"
+      time: "09:00"
+      timezone: "UTC"
+    open-pull-requests-limit: 2
+    groups:
+      js-all-updates:
+        patterns:
+          - "*"
+    labels:
+      - "dependencies"
+      - "automated"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)